source/schema/schema-management-upgrade.txt

```diff
@@ -0,0 +1,206 @@
+=========================
+Schema Management Upgrade
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The |bi| version 2.11 release introduces significant improvements to
+stored schema management. The new stored-schema format is
+equivalent to :ref:`DRDL <schema-with-drdl-file>` files. These
+improvements introduce some backwards-breaking changes for certain
+|bi-short| configurations. The following sections outline the upgrade
+path for various |bi-short| configurations.
+
+Standalone Reader Changes
+-------------------------
+
+Use the following table to modify your |bi-short| version 2.10 or
+earlier Standalone Reader configuration to be compatible with version
+2.11 and later: 
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 45 45
+
+   * - 
+     - Version 2.10 and Earlier
+     - Version 2.11 and Later Upgrade Path
+
+   * - Mode
+     - Standalone Reader
+     - Standalone Reader
+
+   * - Behavior
+     - :binary:`~bin.mongosqld` samples data upon startup and holds the
+       schema in memory.
+     - :binary:`~bin.mongosqld` samples data upon startup and holds the
+       schema in memory.
+
+   * - Options
+     - 
+       - ``--sampleMode read``
+
+       - ``--sampleSource`` *not used*
+
+       .. include:: /includes/fact-v210-schema-options.rst
+
+     - None. The ``--sampleMode`` option is deprecated in version 2.11. Remove
+       this command line option or configuration file setting to
+       continue using standalone mode.
+
+       Any other standalone mode configurations that worked in earlier
+       releases will continue to work with version 2.11.below. 
```

Comment:
Should that be "version 2.11 and later"?

Reply:
Yeah, good catch. I don't know what happened there lol